argocd/quarkus-template/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
```

Comment:
HPA might go a bit beyond a minimal template. Perhaps we should discuss a bit as a team.

Reply:
removed hpa manifest